app/models/grant_submission.rb

```diff
@@ -80,6 +80,9 @@ def self.granted_funding_dollars_total(id_list, query)
     grant_submissions = GrantSubmission.where(id: id_list).where(query).sum(:granted_funding_dollars)
   end
 
+  def funding_levels_csv
```

Comment:
funding_levels_csv is a property of Grant, and shouldn't be a function here.  If you needed to do this, something else is broken.

Reply:
oh shit, sorry that was sloppy. I'm still working through that one; my go-to for starting on a no method error is writing a dummy method.